src/java.base/share/classes/jdk/internal/access/JavaLangAccess.java

```diff
@@ -317,6 +317,18 @@ public interface JavaLangAccess {
      */
     int countNonZeroAscii(String s);
 
+    /**
+     * Constructs a new {@code String} with the supplied Latin1 bytes.
+     * <p>
+     * <b>WARNING: The caller of this method shall relinquish and transfer the
+     * ownership of the byte array to the callee</b>, since the latter will not
+     * make a copy.
+     *
+     * @param bytes the byte array source
+     * @return the newly created string
+     */
+    String uncheckedNewStringWithLatin1Bytes(byte[] bytes);
```

Comment:
@RogerRiggs, we introduce a new `JLA::uncheckedNewStringNoRepl` specialization that
 doesn't throw `CCE` and fix the charset to Latin-1. _"The only reason"_ we do this is to save the call-site from catching `CCE`, am I right?

Reply:
Using `uncheckedNewStringNoRepl` for the performance side effect is confusing, especially with the meaningless try-catch and the constant encoding. A dedicated API allows us to track this particular usage better.